Adjust `Code` to match the actual browsers

CryZe opened the pull request:
This adds the missing `F13` - `F24` values, adds aliases for various values that are still used in Firefox and Safari and adds some values that are only used by Chromium.

Resolves #12
Resolves #13
Resolves #14

pyfisch commented:
Hi @CryZe I'll review the code shortly. Thanks for your patience.

pyfisch commented on convert.py:
```diff
@@ -127,8 +149,51 @@ def convert_code(text, file):
+            []
+        ])
+
+    for chromium_only in [
```
can you put this list in a separate variable `chromium_key_codes`? Multi-line loop conditions are a bit uncommon.

pyfisch commented on convert.py:
```diff
@@ -169,10 +234,11 @@ def convert_code(text, file):
 if __name__ == '__main__':
     input = requests.get('https://w3c.github.io/uievents-key/').text
-    with open('src/key.rs', 'w') as output:
+    with open('src/key.rs', 'w', encoding='utf-8') as output:
```
good catch

pyfisch reviewed (changes requested):
Extending to F13-F24 is good. The alternatives are useful to interact with browsers and shouldn't cause problems.

pyfisch commented on src/code.rs:
```diff
@@ -552,6 +613,37 @@ impl Display for Code {
+            F21 => f.write_str("F21"),
+            F22 => f.write_str("F22"),
+            F23 => f.write_str("F23"),
+            F24 => f.write_str("F24"),
```
For consistence can you add F13 to F24 to Key as well?

CryZe commented:
I addresses the review comments and also fixed the regex. Apparently they published new versions of both specs and added a new `Required` column that the regex couldn't handle.

CryZe commented:
Thank you. Can you potentially release a 0.6.2 for this? I'd like to integrate these changes into druid which has a lot of TODOs regarding missing keycodes.

pyfisch commented:
Yes, already done. :smile:
